Packs/FireEyeCM/Integrations/FireEyeCM/README.md

```diff
@@ -677,3 +677,6 @@ Returns reports on selected alerts.
 #### Human Readable Output
 
 
+## Known Limitations
+Clicking the **Test** button of the **Integration instance settings** window verifies that the instance configuration is correct.
```

Comment:
please add it also to the detailed description

Reply:
added.